scripts/tso-brute.nse

```diff
@@ -94,32 +94,47 @@ Driver = {
 
     local commands = self.options['key1']
     local always_logon = self.options['key2']
+    local skip = self.options['skip']
     stdnse.debug(2,"Getting to TSO")
     local run = stdnse.strsplit(";%s*", commands)
+    stdnse.verbose(2,"Trying User ID/Password: %s/%s", user, pass)
```

Comment:
For this and other debug/verbose messages about trying specific users and passwords, please run at -d2 and see that brute.lua already contains such debugging messages. If yours have extra info, then go ahead and leave them, but you may want to leave them out to avoid duplicate info.